### PR TITLE
Localization: Update folder structure

### PR DESCRIPTION
**Pull request by MarkusAbtion**
Update the locales folder structure to be more strict.

Like with tests, each app-folder will have a corresponding locales-folder.

Move da.yml and en.yml into locales/helpers
Move devise locale files into locales/gems
Initialize locales with new folder structure

### CODE REVIEW

**Comment by runephilosof-abtion** [comment]
You need to update `config/i18n-tasks.yml`

**Comment by runephilosof-abtion** [comment]
We have another project which also has this configuration in `i18n-tasks.yml`

```
data:
  read:
    - config/locales/**/%{locale}.yml
    - config/locales/**/*.%{locale}.yml

  write:
    - ['{devise, kaminari, apipie, pundit}.*', 'config/locales/gems/\1.%{locale}.yml']
    - ['*', 'config/locales/\1/%{locale}.yml']

  yaml:
    write:
      line_width: -1 # do not wrap lines at 80 characters
```

I have not investigated what it does. We have some projects without it.
@prrrnd do you know if we would want it here?

File: Gemfile [hunk 1]
```diff
@@ -7,14 +7,14 @@ ruby "3.0.2"
...
 gem "bootsnap", ">= 1.4.4", require: false
-gem "cancancan"
 gem "devise"
 gem "devise-i18n"
 gem "jquery-rails"
 gem "jsonapi-serializer"
 gem "pg", "~> 1.0"
 gem "pry-rails"
 gem "puma", "~> 5.4"
+gem "pundit"
```
Comment (bohme) [inline Gemfile:17]: this does not belong in this PR 🤔

Comment (MarkusAbtion) [reply Gemfile:17]: You're right. 
I don't know how I merged the two

**Comment by MarkusAbtion** [comment]
> You need to update `config/i18n-tasks.yml`

What do I need to add? 
Just the `ignore_unused` part from https://github.com/abtion/bladkompagniet?

**Comment by MarkusAbtion** [comment]
Ok. I'm not good at handling the auto-merge setup. Didn't mean to merge it yet.

**Comment by runephilosof-abtion** [comment]
In i18n-tasks.yml in muffi, it says:
```
  read:
    ## Default:
    - config/locales/%{locale}.yml
    ## More files:
    - config/locales/**/*.%{locale}.yml
```

I believe it should be to following, in order to read for instance `config/locales/views/da.yml`
```
data:
  read:
    - config/locales/**/%{locale}.yml
    - config/locales/**/*.%{locale}.yml
```

**Comment by runephilosof-abtion** [comment]
I have pushed the change i suggested to main.
